x-pack/filebeat/input/httpjson/encoding.go

```diff
@@ -217,3 +224,13 @@ func decodeAsZip(p []byte, dst *response) error {
 
 	return nil
 }
+
+func decodeAsXML(p []byte, dst *response) error {
+	cdata, v, err := xml.Unmarshal(bytes.NewReader(p), dst.xmlDetails)
+	if err != nil {
+		return err
```

Comment:
Do you think we need an error log here before returning it?

Reply:
No I don't think so, we don't for any of the others and it gets logged in the processor loop logging in `processAndPublishEvents`.